liferay-echo/liferay-portal
# [TS] LPS-196172 Add new error message NoSuchClassTypeException

Motivation
=======
Exporting and Importing a Display Page Template to a new environment via zip file does not give a meaningful error message to the user
https://liferay.atlassian.net/browse/LPS-196172

Proposed Solution
========
Create a new Error Message for com.liferay.asset.kernel.NoSuchClassTypeException

Steps to Verify
========

1. Create a new Web Content Structure S1.
2. Create a new display page template, based on the Blank master, for Web Content, subtype: S1 
3. Publish this template and then export it.
4. Start a fresh bundle
5. Go to display page templates and try to import the file you obtained at step 4.

**Expected behaviour:** An error message is displayed because the Display Page Template’s Subtype can’t be found in this instance.
**Actual behaviour:** An error message is displayed noting that there is a Display Page Template name collision.

Cheers,
Balázs
